README.md

```diff
@@ -20,6 +20,20 @@ The format of the reports is currently either text or HTML.
 
 ## Usage
 
+There is a Dockerfile and a Makefile supplied in this repo. All you need to do is:
+
+1. Build the container image:
+  ```
+  $ make build
+  ```
+
+2. Get shell in the container abd play with the tool:
+  ```
+  $ make shell
+  [root@bd9c05062b1a src]# java -jar target/koji-build-finder-1.0.0-SNAPSHOT-jar-with-dependencies.jar
+  ```
```

Comment:
I had to run `chcon -Rt svirt_sandbox_file_t <pwd>` in order to get the mount working correctly.

Reply:
I always forget the :Z flag, silly me.